api/python/quilt3/main.py

```diff
@@ -465,6 +471,15 @@ def create_parser():
         action="store_true",
         help="Skip the push if the local package hash matches the remote hash.",
     )
+    optional_args.add_argument(
+        "--browse",
+        help="FIXME", # FIXME
```

Comment:
```suggestion
        help="By default, `push` first browses the top_hash from the 'local' registry.  Specify 'remote' to explicitly tell push to retrieve the current top_hash from the destination registry, so that it always succeeds.",
        default="local",
        ```